lib/src/components/NxTabs/NxTabs.scss

```diff
@@ -39,6 +39,7 @@
   // subtract border thickness for consistent spacing
   padding: $nx-spacing-md $nx-spacing-l ($nx-spacing-md - 2px) $nx-spacing-l;
   text-align: center;
+  user-select: none;
```

Comment:
Added to the wrong file?

Reply:
> Added to the wrong file?

Tabs, tags, who can tell the difference? https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/5d18a0512608b8b397746e01615f2da84e26911c